Resources/Public/Configuration/LocalConfiguration.php

```diff
@@ -0,0 +1,177 @@
+<?php
```

Comment:
1. This is the wrong location. An `.php`-file may be executed in `Resources/Public/*`. This is not the right place.
2. I would strip it down to the relevant part: the settings for dlf.
3. I would'nt name it `.php`-file at the end.